common/statsd/statsd.cc

```diff
@@ -69,10 +69,9 @@ class StatsdClientWrapper {
     }
     void timing(const CounterImpl::Timing &tim) { // type: ms
         auto nanoseconds = (tim.end.usec - tim.start.usec) * 1'000;
-        if (tim.tags != nullptr) {
-            addMetric(absl::StrCat(tim.measure, ".duration_ns"), nanoseconds, "ms",
-                      *tim.tags); // format suggested by #observability (@sjung and @an)
-        }
+        // format suggested by #observability (@sjung and @an)
+        addMetric(absl::StrCat(tim.measure, ".duration_ns"), nanoseconds, "ms",
+                  tim.tags == nullptr ? (vector<pair<const char *, const char *>>{}) : *tim.tags);
```

Comment:
I think we wanted to not construct a vector here on default path.
Your call

Reply:
This is only when reporting the metrics, and the code inside addMetric is kind of fiddly so I think I'll leave it. Thanks for the suggestion though.